### PR TITLE
Add vue-vr to WebGL category

### PR DESCRIPTION
**Pull request by imudin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,8 @@
 
 > A curated list of awesome things related to Vue.js
 
+  - [Awesome Vue.js ![Awesome](https://github.com/sindresorhus/awesome)](#awesome-vuejs-awesomehttpsgithubcomsindresorhusawesome)
```
Comment (kazupon) [inline README.md]: Why do you add these?

File: README.md [hunk 2]
```diff
@@ -22,12 +24,12 @@
     - [Examples](#examples)
     - [Books](#books)
 - [Projects Using Vue.js](#projects-using-vuejs)
-  - [Open Source](#open-source)
-  - [Commercial Products](#commercial-products)
-  - [Apps/Websites](#appswebsites)
-  - [Interactive Experiences](#interactive-experiences)
-  - [Enterprise Usage](#enterprise-usage)
-- [Components & Libraries](#components--libraries)
+    - [Open Source](#open-source)
```
Comment (kazupon) [inline README.md]: Why do you adjust?

File: README.md (hunk 1 above)
Comment (imudin) [reply README.md]: There is something wrong, I only added the vue-vr. let me try again

**Comment by imudin** [comment]
Sorry It was cuz of my text editor, it prettified on save. now I rollback and added only one line changes

File: README.md (hunk 1 above)
Comment (imudin) [reply README.md]: I think it is fixed now
